### PR TITLE
HOCS-5561: Use Helm over KD

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -9,28 +9,21 @@ trigger:
     - main
 
 steps:
-  - &deploy
-    name: cs-dev
-    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+  - name: cs-dev
+    image: quay.io/ukhomeofficedigital/helm:v3.5.4
```
Comment (LiamMacP) [inline .drone.yml:13]: This image has a fair amount of vulnerabilities - are they planning on upgrading it?

Comment (nickmcmahon01) [reply .drone.yml:13]: I think we can make some PRs against it, but importantly they should track the kubectl version for us as they update the clusters, there is a drift policy so its not so critical (and it's only drone)

Comment (nickmcmahon01) [reply .drone.yml:13]: https://kubernetes.io/releases/version-skew-policy/

File: .drone.yml [hunk 2]
```diff
@@ -41,22 +34,21 @@ trigger:
...
     - name: checkout
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+      image: alpine/git
       commands:
         - git fetch --tags
         - git checkout $${VERSION}
 
     - name: prod deploy
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+      image: quay.io/ukhomeofficedigital/helm:v3.5.4
```
Comment (nickmcmahon01) [inline .drone.yml:43]:
```suggestion
      image: quay.io/ukhomeofficedigital/helm
```

File: .drone.yml (hunk 1 above)
Comment (nickmcmahon01) [inline .drone.yml:13]:
```suggestion
    image: quay.io/ukhomeofficedigital/helm
```

File: .drone.yml [hunk 3]
```diff
@@ -66,17 +58,16 @@ steps:
 
     - &deploy
       name: notprod deploy
-      image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+      image: quay.io/ukhomeofficedigital/helm:v3.5.4
```
Comment (nickmcmahon01) [inline .drone.yml:61]:
```suggestion
      image: quay.io/ukhomeofficedigital/helm
```

**Review by SWhitlam — APPROVED** [review]

**Pull request merged**